src/components/Button/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {ButtonSize} from '@gravity-ui/uikit';
+
+import {ButtonImagePosition, ButtonTheme} from '../../../models';
+import {ICON_QA} from '../utils';
+
+import Button from '../Button';
+
+const qaId = 'button-component';
+
+const buttonProps = {
+    text: 'Button Text',
+    url: 'https://github.com/gravity-ui/',
+    target: '_blank',
+    img: {
+        url: 'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-gray.png',
```

Comment:
it is image for the button? Why is it not icon?

Reply:
Sure